app/views/owner/experiences/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="container px-5">
+  <div class="container px-5">
+    <div class="d-flex justify-content-between py-5">
```

Comment:
I believe you need `align-items-center` here otherwise it looks like this:
<img width="169" alt="Screen Shot 2022-01-17 at 10 07 41" src="https://user-images.githubusercontent.com/25542223/149686011-84496fa6-b1e2-4dfd-9507-ed6c7c5f4e9c.png">

Reply:
yours looks different than mind 🤔